lib/css/CssLoadingRuntimeModule.js

```diff
@@ -163,13 +163,9 @@ class CssLoadingRuntimeModule extends RuntimeModule {
 			crossOriginLoading
 				? crossOriginLoading === "use-credentials"
 					? 'link.crossOrigin = "use-credentials";'
-					: Template.asString([
-							"if (link.href.indexOf(window.location.origin + '/') !== 0) {",
-							Template.indent(
-								`link.crossOrigin = ${JSON.stringify(crossOriginLoading)};`
-							),
-							"}"
-						])
+					: `link.crossOrigin = ${JSON.stringify(
```

Comment:
Can be further simplified to remove the ternary
Can make that change if we want to continue with this PR